webapp/src/ts/modules/reports/reports.routes.ts

```diff
@@ -1,20 +1,20 @@
 import { Routes } from '@angular/router';
 
-import { RouteGuardProvider } from '@mm-providers/route-guard.provider';
+import { AppRouteGuardProvider } from '../../app-route.guard.provider';
```

Comment:
:( I'm sad the alias is gone and we now have to use a relative path to the provider.

Reply:
I thought on adding an alias for `webapp/src` like `@mm-app` or `@mm-src`
I have to test that it doesn't mess up the IDE suggestions, like `@mm-src/components/my-component` instead of `@mm-components/my-component`